wayfinder/manifest.json

```diff
@@ -0,0 +1,65 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "a68bad83-ba55-4350-a913-2f98bb667bad",
+  "app_id": "wayfinder",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send Wayfinder metrics to Datadog",
+    "title": "Appvia Wayfinder",
+    "media": [
+      {
+        "media_type": "image",
```

Comment:
Please ensure that images meet our [media requirements](https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#media-carousel), namely all images have to have a 16:9 aspect ratio.

Reply:
Also, once the dashboard is updated, please include a new screenshot - thanks!